Chapter9/Chapter9/View/RestaurantDetailMapCell.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  RestaurantDetailMapCell.swift
+//  Chapter9
+//
+//  Created by Nazih Al Tajar on 02/04/2019.
+//  Copyright © 2019 sss. All rights reserved.
+//
+
+import UIKit
+import MapKit
+
+class RestaurantDetailMapCell: UITableViewCell {
+
+    @IBOutlet var mapView: MKMapView!
+
+    func configure(location: String) {
+        let geoCoder = CLGeocoder()
+
```

Comment:
In a class above you didn't leave any space between these two. Or you put extra space in the both cases or no.